content/backend/graphql-scala/0-introduction.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Introduction
+pageTitle: "Building a GraphQL Server with Scala Backend Tutorial"
+description: "Learn how to build a GraphQL server with Scala & Sangria and best practices for filters, authentication, pagination and subscriptions. "
+
+---
+
+### Motivation
+
+Scala is very popular language nowadays and it's often chosen to deliver efficient and distributed systems. It leverages the Java VM, known mostly for its efficiency. Support of Functional Programming and power of Java make able to deliver applications fast in either development or runtime.
+
```

Comment:
Consider a short glue sentence, for instance why GraphQL is good pick for Scala. Scala is mostly backend, used for complex systems to you probably want to focus on the big things not busy work like APIs ... etc.